### DISCUSSION TITLE
Attempt at fixing package corruption

### DISCUSSION THREAD

**Discussion by soniamitchell** [IBAHCM/RPiR#160] [Archive] [answered]
Hey @shayavh, if you have time could you do me a favour and try to reinstall the RPiR package, v0.52.5? and check whether you can run multiple practicals in a row? I'm trying to isolate the problem. 

```
install_github("IBAHCM/RPiR")
```

**Comment by shayavh** [comment, marked as answer]
Hi @soniamitchell, it took quite a while for it to install the package, but you solved it! All the practicals work now without any reinstalling or restarting of R. Thanks

**Reply by soniamitchell** [reply]
Wooooooo!!!! 🎉🎉 🎉
